.gitignore

```diff
@@ -5,3 +5,4 @@ bootstrap.mak
 /runtimes.mak
 metadata.min.js
 /rfc8794.xml
+/.vs
```

Comment:
This is not referenced anywhere in our code. (you can add it to your local gitignore instead)

Reply:
fixed